tutorials/calibration_flow/static_quant.py

```diff
@@ -113,46 +146,99 @@ def forward(self, x):
         x = self.linear2(x)
         return x
 
-torch.manual_seed(0)
+def test_intx():
```

Comment:
optional: maybe test code can be merged as well?

Reply:
The call to quantize_() and AffineQuantizedMinMaxObserver() have different param values for floatx and intx. I've merged it and also added an assert for Mapping_type as float8 supports only Symmetric